File: CS50x/Weeks 1-5/Week 1/mario/mario.c
#include <cs50.h>
#include <stdio.h>
int main(void)
{
    int height;
    do
    {
         height = get_int("What is the height? ");
    }
    while (height < 1 || height > 8);


    for (int i = 0; i < height; i++)
    {
        //generate H-i spaces per row
        for( int spaces = 0; spaces < height-i; spaces++)
        {
            printf(" ");
        }
        //generate i hashes per row
        for( int l_hashes = 0; l_hashes <= i; l_hashes++)
        {
            printf("#");
        }
        //print 2 spaces
        printf("  ");

        //print i hashes again
        for( int r_hashes = 0; r_hashes <=i; r_hashes++)
        {
            printf("#");
        }

        //new line
        printf("\n");
    }

}

File: CS50x/sentimental-mario-more/mario.py
from cs50 import get_string

height = 0
while height <1 or height>8:
    height = int(get_string("Height:"))

for i in range(height):
    print((" " * (height - i)) + ("#" * i) + " " + ("#" * i) + " " * (height - i) )

